corehq/apps/app_manager/app_schemas/app_case_metadata.py

```diff
@@ -57,9 +57,15 @@ def _add_module_contributions(self):
 
     def _add_module_contribution(self, module):
         for column in module.case_details.long.columns:
-            self.meta.add_property_detail('long', module.case_type, module.unique_id, column)
+            self.meta.add_property_detail('long', module.case_type, module.unique_id,
+                                          column.field, column.header, column.useXpathExpression)
         for column in module.case_details.short.columns:
-            self.meta.add_property_detail('short', module.case_type, module.unique_id, column)
+            self.meta.add_property_detail('short', module.case_type, module.unique_id,
+                                          column.field, column.header, column.useXpathExpression)
+        if module.case_details.short.filter:
+            header = {self.app.default_language: _("[Filter]")}     # Fake a localized "header"
```

Comment:
What happens if you are looking at a different language than the default? Does it just pick this one?

Reply:
I'm guessing it'll prepend a label, so it'd appear like `[en] [Filter]` if the default language was English and you viewed it in another language. Which is weird. Added 55c64a41510 so it'll display the same in all languages.